src/parser/on_demand/parse_to_structure.ts

```diff
@@ -0,0 +1,138 @@
+import { type Code } from '../../code';
+import { type BSONElement, getSize, parseToElements as p } from './parse_to_elements';
+
+/** @internal TODO */
+const DEFAULT_REVIVER = () => null;
+
+/** @internal */
+function parseToElements(...args: Parameters<typeof p>): BSONElement[] {
```

Comment:
What's the logic behind having a parseToElements wrapper in parse_to_structure file with the same name as parseToElements in the ./parse_to_elements file? The naming seems a bit confusing to me at first glance.

Reply:
Fixed, just a stylistic choice to not create something with a new name for a small piece of logic. This works too.